third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -700,6 +722,18 @@ func resourceComposerEnvironmentUpdate(d *schema.ResourceData, meta interface{})
 				return err
 			}
 		}
+<% end -%>
```

Comment:
Probably better to combine these two conditions since they're identical, here and elsewhere.